source/frontend/src/features/mapSideBar/acquisition/models/DetailAcquisitionFileOwner.ts

```diff
@@ -0,0 +1,89 @@
+import { Api_AcquisitionFileOwner } from '@/models/api/AcquisitionFile';
+import { Api_Address } from '@/models/api/Address';
+
+export class DetailAcquisitionFileOwner {
```

Comment:
ok, this confused the heck out of me but looks like just a file move from previous story.

Reply:
moved some models to a higher folder to be able to share with other files.